test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -173,36 +192,66 @@ void test_ucp_ep_reconfig::entity::store_config()
     }
 
     m_cfg_index = ep()->cfg_index;
-}
 
-ucp_tl_bitmap_t test_ucp_ep_reconfig::entity::ep_tl_bitmap() const
-{
-    if (ep() == NULL) {
-        return UCS_STATIC_BITMAP_ZERO_INITIALIZER;
+    /* Calculate number of reused resources by:
+     * 1) Count number of resources used in EP configuration.
+     * 2) Take half of total resources to be reused. */
+    ucp_tl_bitmap_t rscs_bitmap = UCS_STATIC_BITMAP_ZERO_INITIALIZER;
+
+    for (auto lane = 0; lane < ucp_ep_num_lanes(ep()); ++lane) {
+        UCS_STATIC_BITMAP_SET(&rscs_bitmap, ucp_ep_get_rsc_index(ep(), lane));

```

Comment:
seems we could replace this loop by calling ep_tl_bitmap with unlimited max_num_rscs